### DISCUSSION TITLE
generate-blocklists.py

### DISCUSSION THREAD

**Discussion by privacyguy123** [DNSCrypt/dnscrypt-proxy#2002] [Potential issues]
Is there an issue with python not understanding the list with wildcards @ https://dblw.oisd.nl/ ? This is the one they even recommend for use with dnscrypt. When I try to generate from that URL my text file shows blank.

**Comment by jedisct1** [comment]
Hi,

Names with wildcards are indeed skipped, as this is not a common thing in public lists. But this is something that should be easy to add.

I guess what they suggest is using that list directly, not through the Python script.

**Comment by jedisct1** [comment]
f7e33816508da5f03c0ca373bfb5a26c4cd6c1c4 adds supports for this.
